pkg/internal/checkup/checkup.go

```diff
@@ -332,3 +366,7 @@ func trafficGenName(suffix string) string {
 func trafficGenConfigMapName(suffix string) string {
 	return TrafficGenConfigMapNamePrefix + "-" + suffix
 }
+
+func vmiUnderTestConfigMapName(suffix string) string {
+	return vmiUnderTestConfigMapNamePrefix + "-" + suffix
+}
```

Comment:
This function is a duplicate of `trafficGenConfigMapName()`, please consider adjusting the signature of `trafficGenConfigMapName` and using a single function for both uses.

Reply:
Let's do it in a later PR